ip2location.c

```diff
@@ -24,6 +24,18 @@
 
 #include "php_ip2location.h"
 
+#if PHP_MAJOR_VERSION >= 8
+#include "ip2location_arginfo.h"
+
+#else
+/* For PHP < 5.3.7 */
```

Comment:
Do you really want to keep compatibility with such old thing ?  (package.xml states "5.0.0")

Reply:
Can you remove this? Thanks.

Reply:
done